components/console/src/components/form/search-account-by-alias-field/index.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useState, useCallback } from 'react'
+import { Controller, Control } from 'react-hook-form'
+import { useIntl } from 'react-intl'
+import { Label } from '@/components/ui/label'
+import { Input } from '@/components/ui/input'
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/ui/popover'
+import { Loader2 } from 'lucide-react'
+import { useDebounce } from '@/hooks/use-debounce'
+import { useListAccounts } from '@/client/accounts'
+import { useOrganization } from '@lerianstudio/console-layout'
+import { FormTooltip } from '@/components/ui/form'
+import { AccountDto } from './types'
+
+export interface SearchAccountByAliasFieldProps {
+  control: Control<any>
+  name: string
+  label?: string
+  tooltip?: string
+  placeholder?: string
+  required?: boolean
+  disabled?: boolean
+  rules?: any
+  maxSuggestions?: number
+  debounceDelay?: number
+  className?: string
+  onAccountSelect?: (account: AccountDto) => void
+  onSearchChange?: (searchTerm: string) => void
+}
+
+export const SearchAccountByAliasField: React.FC<SearchAccountByAliasFieldProps> = ({
+  control,
+  name,
+  label,
+  tooltip,
+  placeholder,
+  required = false,
+  disabled = false,
+  rules,
+  maxSuggestions = 5,
+  debounceDelay = 600,
+  className = '',
+  onAccountSelect,
+  onSearchChange
+}) => {
+  const intl = useIntl()
+  const { currentOrganization, currentLedger } = useOrganization()
+  const [searchTerm, setSearchTerm] = useState('')
+  const [debouncedSearchTerm, setDebouncedSearchTerm] = useState('')
+
+  const debouncedSearchCallback = useCallback(() => {
+    setDebouncedSearchTerm(searchTerm)
+  }, [searchTerm])
+
+  useDebounce(debouncedSearchCallback, debounceDelay, [searchTerm])
+
+  const { data: accountsData, isLoading: accountsLoading, error: accountsError } = useListAccounts({
+    organizationId: currentOrganization?.id || '',
+    ledgerId: currentLedger?.id || '',
+    query: debouncedSearchTerm ? { alias: debouncedSearchTerm } : undefined,
+    enabled: !!currentOrganization?.id && !!currentLedger?.id && !!debouncedSearchTerm
```

Comment:
Avoid using enabled.
This property disabled the invalidateQuery functionality from react-query.